contents/handbook/company/diversity.md

```diff
@@ -53,7 +53,7 @@ We don't just state that we care - these are some of the things we've implemente
 * [Paid SuperDay](/careers#the-process) as part of the hiring process - to allow you to see what it's really like working on the team, before starting.
 * [Training budget](/handbook/people/training#training-budget) for those in roles where we don't have lots of existing experience as a company.
 * Life story Fridays (when we have a new team member, we'll ask them to present their life story for an hour on a Friday) - so you have more context on the points of views of others in the team.
-* [Sponsored visas](/handbook/people/hiring-process#visa-sponsorship) for those who need them
+* [Sponsored visas](/handbook/people/hiring-process#visa-sponsorship) for those who need them.
```

Comment:
All other items in this list end in a period. Adding a period here for consistency.